lua/lvim/lsp/config.lua

```diff
@@ -64,12 +64,11 @@ return {
       header = "",
       prefix = "",
       format = function(d)
-        local t = vim.deepcopy(d)
```

Comment:
the `deepcopy()` was necessary to avoid messing up the original diagnostics, are you sure it's not needed anymore?

Reply:
we're not mutating anymore though, instead of changing the field we just simply return.

Reply:
nice catch!